setup.php

```diff
@@ -143,13 +143,24 @@ function cloneClicked() {
 <span class="text">
 
 <?php
+// Checks if the server's PHP version and the client's browser are compatible with OpenEMR:
+require_once(dirname(__FILE__) . "/common/compatibility/checker.php");
```

Comment:
As an aside, can we just include the composer autoloader script on these scripts(setup, upgrades, and admin) without context?